pysr/sr.py

```diff
@@ -1013,6 +1013,22 @@ def from_file(
             base_equation_file = os.path.basename(model.equation_file_)
             model.equation_file_ = os.path.join(base_dir, base_equation_file)
 
+            # Get constructor parameters and default values
+            params = inspect.signature(model.__init__).parameters
+
+            # Filter for missing parameters excluding kwargs
+            missing_params = {
+                k: v
+                for k, v in params.items()
+                if k not in model.__dict__.keys()
+                and v.name != "self"
+                and v.kind != v.VAR_KEYWORD
+            }
+
+            # Assign missing attributes
+            for k, v in missing_params.items():
+                setattr(model, k, v)
```

Comment:
Could we also have this print a warning that certain parameters are being set to defaults?

Reply:
I've added one